cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -532,6 +555,10 @@ fn x64_get_regs(inst: &Inst, collector: &mut RegUsageCollector) {
             src.get_regs_as_uses(collector);
             collector.add_mod(*dst);
         }
+        Inst::XMM_MOV_RM_R { op: _, src, dst } => {
```

Comment:
(I've got a large patch that makes this more uniform, so let's do it here too) Can you use this syntax to omit the unneeded fields: `XMM_MOV_RM_R { src, dst, .. } => `